Source/CarthageKit/Xcode.swift

```diff
@@ -1220,9 +1236,17 @@ public func buildInDirectory(directoryURL: NSURL, withConfiguration configuratio
 							return false
 						}
 					}
-					.concat(SignalProducer(error: .NoSharedFrameworkSchemes(.Git(GitURL(directoryURL.path!)), platforms)))
-					.take(1)
-					.flatMap(.Merge) { project, schemes in SignalProducer(values: schemes.map { ($0, project) }) }
+					.flatMap(FlattenStrategy.Merge) { project, schemes in
```

Comment:
I think it'd be clearer to keep the `.Concat` strategy here since the order could be considered important.